pos.lua

```diff
@@ -41,13 +41,13 @@ minetest.register_chatcommand("select_area", {
 })
 
 minetest.register_chatcommand("area_pos1", {
-	params = "[X Y Z|X,Y,Z]",
+	params = "[X Y Z|X,Y,Z|X, Y, Z]",
 	description = S("Set area protection region position @1 to your"
 		.." location or the one specified", "1"),
 	privs = {},
 	func = function(name, param)
 		local pos
-		local found, _, x, y, z = param:find(
+		local found, _, x, y, z = param:gsub(", ", ","):find(
 				"^(-?%d+)[, ](-?%d+)[, ](-?%d+)$")
```

Comment:
Suggestion: instead of additional processing, fix the regular expression.
This here works: `"^(-?%d+)[, ] *(-?%d+)[, ] *(-?%d+)"`

FYI: The Minetest builtin commands parse it like this: https://github.com/minetest/minetest/blob/master/builtin/game/chat.lua#L584
Those also support relative positions, which however is not a feature we need in this case (player position is too unrealiable).